src/components/Input/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,31 @@
+import { cx } from '@emotion/css';
+import React, { PropsWithChildren } from 'react';
+import { TextAreaProps } from './TextAreaProps';
+import { textAreaStyle, stySizeDefault, styStatusDefault } from './styles';
+
+const TextArea = (props: PropsWithChildren<TextAreaProps>): JSX.Element => {
```

Comment:
```suggestion
const TextArea = (props: PropsWithChildren<TextAreaProps>) => {
```
sebenernya tuh typescript pinter enough to know that youre returning a JSX.Element..

unnecessary things clutters up the code, isn't it?